File: go-sdk/pkg/models/operations/deleteweblinksid.go
package operations

import (
	"openapi/pkg/models/shared"
	"openapi/pkg/utils"
)

type DeleteWebLinksIDPathParams struct {
	WebLinkID string `pathParam:"style=simple,explode=false,name=web_link_id"`
}

type DeleteWebLinksIDRequest struct {
	Retries    *utils.RetryConfig
	PathParams DeleteWebLinksIDPathParams
}

type DeleteWebLinksIDResponse struct {
	ClientError *shared.ClientError
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/deleteweblinksid.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class DeleteWebLinksIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=web_link_id" })
  webLinkId: string;
}


export class DeleteWebLinksIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;

  @SpeakeasyMetadata()
  pathParams: DeleteWebLinksIdPathParams;
}


export class DeleteWebLinksIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
